sys/fido2/ctap/ctap_mem.c

```diff
@@ -62,12 +82,34 @@ ctap_status_code_t fido2_ctap_mem_read(void *buf, uint32_t page, uint32_t offset
     return CTAP2_OK;
 }
 
-static ctap_status_code_t _flash_write(const void *buf, uint32_t page, uint32_t off, size_t len)
+static bool _flash_rk_check_offset(uint32_t off)
+{
+    /* Offset must skip ctap_state_t */
+    if (off < CTAP_FLASH_STATE_SZ) {
+        DEBUG("%s, %d: Incorrect offset detected\n", RIOT_FILE_RELATIVE,
```

Comment:
```suggestion
        DEBUG("%s:%d: Incorrect offset detected\n", __FILE__,
```

would make the link clickable in some terminals. `RIOT_FILE_RELATIVE` has actually been deprecated way before it was documented, see https://github.com/RIOT-OS/RIOT/pull/21170#issuecomment-2682939049

Same below.